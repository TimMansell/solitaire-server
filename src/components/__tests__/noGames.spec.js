import { shallowMount } from '@vue/test-utils';
import NoGames from '@/components/NoGames.vue';

describe('NoGames.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(NoGames);

    expect(wrapper).toMatchSnapshot();
  });
});
