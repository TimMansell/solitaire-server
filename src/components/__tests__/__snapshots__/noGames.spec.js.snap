// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoGames.vue matches snapshot 1`] = `<p>You have not played any games yet</p>`;
